sqs/Module.md

```diff
@@ -92,300 +34,20 @@ sqs:Configuration configuration = {
-    }
-
-    // Send a message to the created queue
-    attributes = {};
-    attributes["MessageAttribute.1.Name"] = "Name1";
-    attributes["MessageAttribute.1.Value.StringValue"] = "Value1";
-    attributes["MessageAttribute.1.Value.DataType"] = "String";
-    attributes["MessageAttribute.2.Name"] = "Name2";
-    attributes["MessageAttribute.2.Value.StringValue"] = "Value2";
-    attributes["MessageAttribute.2.Value.DataType"] = "String";
-    string queueUrl = "";
-    sqs:OutboundMessage|error response2 = sqsClient->sendMessage("Sample text message.", queueResourcePath,
-        attributes);
-    if (response2 is sqs:OutboundMessage) {
-        log:printInfo("Sent message to SQS. MessageID: " + response2.messageId);
-    }
-
-    // Receive a message from the queue
-    string[] attributeNames = ["SenderId"];
-    string[] messageAttributeNames = ["Name2"];
-    sqs:InboundMessage[]|error response2 = sqs->receiveMessage(standardQueueResourcePath, 10, 2, 1);
-    if (response3 is sqs:InboundMessage[] && response3.length() > 0) {
-        log:printInfo("Successfully received the message. Message body: " + response3[0].body);
-        log:printInfo("\nReceipt Handle: " + response3[0].receiptHandle);
-        // Keep receipt handle for deleting the message from the queue
-        receivedReceiptHandler = response3[0].receiptHandle;
-    }
-
-    // Delete the received the message from the queue
-    error? response4 = sqsClient->deleteMessage(queueResourcePath, receivedReceiptHandler);
-    if (response4 is ()) {
-        log:printInfo("Successfully deleted the message from the queue.");
-    }
-
-    // Delete the queue
-    error? response5 = sqsClient->deleteQueue(queueResourcePath);
-    if (response is ()) {
-        log:printInfo("Successfully deleted the queue.");
-    }
-}
-```
-
-## Managing SQS FIFO Queue
-
-This example describes how a SQS FIFO Queue is created, a message is sent to it, received from the queue, and deleted from the queue. 
-
-```ballerina
-import ballerina/log;
-import ballerinax/aws.sqs;
-
-configurable string accessKeyId = ?;
-configurable string secretAccessKey = ?;
-configurable string region = ?;
-configurable string accountNumber = ?;
-
-public function main(string... args) {
-
-    // Add the SQS credentials as the Configuration
-    sqs:Configuration configuration = {
-        accessKey: accessKeyId,
-        secretKey: secretAccessKey,
-        region: region,
-        accountNumber: accountNumber
-    };
-
-    sqs:Client sqsClient = check new (configuration);
+### Step 3: Invoke connector operation
 
-    // Declare common variables
-    string queueResourcePath = "";
-    string receivedReceiptHandler = "";
+1. You can create a queue in SQS as follows with `createQueue` method for a preferred queue name and the required set of attributes. Successful creation returns the created queue URL as a string and the error cases returns an `error` object.
```

Comment:
Change "error cases returns an `error` object." to "error cases return an `error` object.".

If this is going to be a substep, use the imperative tone.